modules/unsupported/geopkg/src/test/java/org/geotools/geopkg/GeoPkgDataStoreAPITest.java

```diff
@@ -0,0 +1,88 @@
+package org.geotools.geopkg;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.geotools.data.DataStore;
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder;
+import org.geotools.referencing.CRS;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+import com.vividsolutions.jts.geom.Point;
+
+public class GeoPkgDataStoreAPITest {
+
+    DataStore dataStore = null;
+
+    @Rule
+    public TemporaryFolder tmp = new TemporaryFolder(new File("target"));
+
+    @Before
+    public void setUp() throws Exception {
+        Map<String, Serializable> map = new HashMap<>();
+        map.put(GeoPkgDataStoreFactory.DBTYPE.key, "geopkg");
+        map.put(GeoPkgDataStoreFactory.DATABASE.key, "foo.gpkg");
+
+        GeoPkgDataStoreFactory factory = new GeoPkgDataStoreFactory();
+        factory.setBaseDirectory(tmp.getRoot());
+        dataStore = factory.createDataStore(map);
+    }
+
+    @Test
+    public void testCreateDataTypes() throws Exception {
```

Comment:
Sure, there are some "online" tests and the first intent was to add it there. But, ehm, I didn't manage to run the existing tests! I should have asked in the list...

But in fact, I then preferred to deliver a simpler test which actually can be always run, not depending on an external condition which is difficult to meet and in most cases implies test will be skipped.

As far as I understood, the "online" tests were made optional and needed a configuration file because they depend on external services (namely a running database instance). Geopkg doesn't need such an external service, so their tests don't need to be "online" and can be always run.